calico-cloud/get-started/connect/requirements/system-requirements.mdx

```diff
@@ -36,13 +36,21 @@ kubectl get <type> -n <namespace> <resource-name> -o yaml | grep ' addonmanager.
 
 ### Step 3: Prepare your infrastructure to connect to {{prodname}}
 
-Pods running in your kubernetes cluster should be allowed to connect to the following endpoints:
+Pods running in your Kubernetes cluster must allow outbound traffic to the following endpoints:
 
-- https://installer.calicocloud.io:443/*
-- https://www.calicocloud.io:443/api/*
+- `https://installer.calicocloud.io:443/*`
+- `https://www.calicocloud.io:443/api/*`
 - TCP to `<management-cluster>.calicocloud.io:9000`
 
-Docker running on each node should be able to pull images from `quay.io`,`cdn01.quay.io`,`cdn02.quay.io`, and `us-docker.pkg.dev`
+For each node, Docker must be able to pull images from the following endpoints:
+
+* `quay.io`
+* `cdn01.quay.io`
+* `cdn02.quay.io`
+* `us-docker.pkg.dev`
+
+These connections must be allowed by your Kubernetes cluster and by the rest of your infrastructure.
+Other elements of your infrastructure, such as firewalls or security groups, can restrict access to {{prodname}} endpoints even if your cluster allows the connections.
```

Comment:
I feel like this could confuse people and they could interpret this as "it is ok to restrict access at other places in the infrastructure". I think rewording it to the follow could help:
```
Other elements of your infrastructure, such as firewalls or security groups, might restrict access to {{prodname}} endpoints so ensure the traffic is allowed by these other elements also.
```

Reply:
Thanks, I've adjusted the wording for clarity.